src/native/libs/System.JavaScript/ReadMe.md

```diff
@@ -0,0 +1,15 @@
+# System.JavaScript
+
+Is TypeScript project that is compiled via `src\native\libs\rollup.config.js` into Emscripten JS library.
```

Comment:
Nit: "Is TypeScript project that ...." reads as a start of a question. I would either drop "Is" or make it full sentence "This directory contains...".